media/css/firefox/personal.less

```diff
@@ -0,0 +1,323 @@
+        }
+
+        @media only screen and (max-width: @breakMobile) {
+            min-width: 260px;
+            .font-size(1rem);
+        }
+
+        &:hover {
+            background-color: #7dc14c;
+        }
+        .download-link {
+            box-shadow: none;
+            background-color: inherit;
+            background-image: none;
+            font-family: inherit;
+            font-weight: bold;
+            text-shadow: none;
+        }
+    }
+    .download-content {
+        text-transform: uppercase;
+    }
+}
+main {
+    margin: 0 auto;
+    padding: 60px 0 150px 0;
+    width: 100%;
+    max-width: 1400px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding: 20px 0;
+    }
+}
+.mission {
+    display: inline-block;
+    padding: 0 25px;
+    width: 25%;
+    vertical-align: top;
+
+    @media only screen and (max-width: @breakTablet) {
+        display: block;
+        margin: (@baseLine * 3) 0 0 0;
+        padding: 0;
+        width: 100%;
+    }
+
+    &:first-child {
+        @media only screen and (max-width: @breakTablet) {
+            margin-top: 0;
+        }
+    }
+
+    p {
+        color: #999;
+        font-family: 'FiraSans Light';
+        .font-size(1.75rem);
+        line-height: 1.3;
+
+        @media only screen and (max-width: @breakDesktop) {
+            .font-size(1.35rem);
+        }
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(1.25rem);
+        }
+
+        span {
+            display: block;
+            color: #56565a;
+            .font-size(2rem);
+            font-weight: bold;
+            text-transform: uppercase;
+        }
+    }
+    p:before {
+        display: block;
+        background: transparent url(/media/img/firefox/personal/mission-icons-sprite.png) 50% 0 no-repeat;
+        margin-top: @baseLine;
+        height: 175px;
+        content: '';
+    }
+    p.data:before {
+        background-position: 50% -174px;
+    }
+    p.choice:before {
+        background-position: 50% -380px;
+    }
+    a {
+        .font-size(1.25rem);
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(1.1rem);
+        }
+
+        &:hover {
+            color: #005da5;
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Simple Footer
+#footer {
+    background: #fff;
+
+    .content {
+        margin: 0 auto;
+        padding: (@baseLine * 2) 0 (@baseLine * 3);
+        max-width: 1400px;
+        text-align: center;
+
+        @media only screen and (max-width: @breakTablet) {
+            padding: @baseLine 0;
+            width: 100%;
+        }
+    }
+
+    .mozlogo {
+        .image-replaced();
+        display: inline-block;
+        margin: 0 20px -2px 0;
+        background: url(/media/img/firefox/family/index/family-sprite.png) -345px -485px no-repeat;
```

Comment:
Yup, saw that a little while ago, was not aware it contained all of those assets.